src/context.rs

```diff
@@ -1764,6 +1764,20 @@ macro_rules! symbol_with_update {
   };
 }
 
+pub fn av1_get_adjusted_tx_size(tx_size: TxSize) -> TxSize {
```

Comment:
Is there a better name for this? Or a comment? Like "nonzero_coeffs_size" or something?

Reply:
The adjusted size is what level map coding is seeing, so maybe coded_size then

Reply:
so, av1_get_coded_tx_size()

Reply:
Er, that's also confusing because it's not the tx size we write :/